<p>
  <code>Method</code> must have Javadoc comments..
</p>
<h2>Noncompliant Code Example</h2>
<pre>
// Noncompliant
public class MethodMustHaveJavaDocCommentCheck {
  void test() {}
}
</pre>
<h2>Compliant Solution</h2>
<pre>
/**
 * Detail info
 * @author Shawn
 * @date 2020/7/23
 */
public void test2() {
  // ...
}
</pre>
