<p>
  <code>Class</code> and <code>Interface</code> must have Javadoc..
</p>
<h2>Noncompliant Code Example</h2>
<pre>
// Noncompliant
public class ClassMustHaveJavaDocCheck {
  // ...
}
</pre>
<h2>Compliant Solution</h2>
<pre>
/**
 * Detail info
 * @author Shawn
 * @date 2020/7/23
 */
public class ClassMustHaveJavaDocCheck {
  // ...
}
</pre>
